### ISSUE TITLE
Links The Challenge of Golf (1990)

### ISSUE THREAD

**Issue by Eulisker** [exoscoriae/eXoDOS#3231]
I scanned

- [x] Access - Product Catalog
- [x] CLUB Distances
- [x] Disk Offer
- [x] Note
- [x] Warranty Card

Added to the github folder.

**Comment by Eulisker** [comment]
I scanned a couple of manuals and score cards for the addon courses:

- [ ] Barton Creek - Instruction Manual
- [ ] Barton Creek - Score Card
- [ ] Pinehurst Resort & Country Club - Installation Manual
- [ ] Pinehurst Resort & Country Club - Score Card

I also scanned

- [ ] UK Warranty Information

This was in both of the UK releases i own. I think it is likely in all of the UK Releases. But i doubt it is useful to have this seperately for all the Addons. 
The same goes for the Registration Card for Firestone. This one is a US Release and has the US Registration Card. Maybe it is possible to rename it something like "Championship Course - US Registration Card" and not repeat it for the other addons if and when they are added.

**Comment by Eulisker** [comment]
Same here. I have to reupload on the weekend. Closing this for now.

**Comment by Eulisker** [comment]
I scanned a couple of manuals and score cards for the addon courses:

- [ ] Barton Creek - Instruction Manual
- [ ] Barton Creek - Score Card
- [ ] Pinehurst Resort & Country Club - Installation Manual
- [ ] Pinehurst Resort & Country Club - Score Card
- [ ] Troon North - Score Card


I also scanned

- [ ] UK Warranty Information

This was in both of the UK releases i own. I think it is likely in all of the UK Releases. But i doubt it is useful to have this seperately for all the Addons.
The same goes for the Registration Card for Firestone. This one is a US Release and has the US Registration Card. Maybe it is possible to rename it something like "Championship Course - US Registration Card" and not repeat it for the other addons if and when they are added.

Lastly i scanned 2

- [ ] Envelopes

The Disks and supplemental material of the addon courses came in a sealed envelope inside the box. I found 2 slightly different ones.
